File: Java/Lista_03(Estrutura_Repetitiva)/Senha_Fixa.java
import java.util.Scanner;

public class Senha_Fixa {

	public static void main(String[] args) {

		Scanner s = new Scanner(System.in);
		
		System.out.print("Digite a senha: ");
		int senha = s.nextInt();
		
		while(senha!=2002) {
			System.out.print("Senha invalida! Tente novamente: ");
			senha = s.nextInt();
		}
		
		System.out.print("Acesso permitido!");
		
		s.close();
		
	}

}

File: C++/Lista_03(Estrutura_Repetitiva)/Senha_Fixa.cpp
#include <iostream>

using namespace std;

int main(){

    string senha;

    cout << "Digite a senha: ";
    cin >> senha;

    while(senha!="2002"){
        cout << "Senha invalida! Tente novamente: ";
        cin >> senha;
    }

    cout << "Acesso permitido!";

    return 0;

}